### DISCUSSION TITLE
Refactor assets to serve from specific directory

### DISCUSSION THREAD

**Discussion by kersten** [inovex/scrumlr.io#4724] [Proposals]
Hello,

We currently deploy Scrumlr to our Kubernetes cluster, and we have Kyverno policies in place that prohibit using `ImplementationSpecific` path types in our NGINX ingress configurations, because of some CVEs. At the moment, static files like `timer_finished.mp3` and `hotkeys.pdf` are served directly from the frontend at top-level paths (e.g., `/timer_finished.mp3`). This makes it impossible for us to switch to `Prefix` path types for routing—something that our policies require.

**Request:**  
Please consider moving these static assets under a dedicated directory, for example `/assets/`. For instance:
- `/timer_finished.mp3` → `/assets/timer_finished.mp3`
- `/hotkeys.pdf` → `/assets/hotkeys.pdf`

By serving static content under an `/assets` prefix, we can:
- Use a `Prefix` path route, complying with Kyverno policies.
- Maintain a cleaner and more compliant ingress configuration.

This small change would greatly simplify our deployment process and help us maintain compliance while still serving the required assets.

Thank you for your consideration.

**Comment by Schwehn42** [comment]
Hi @kersten, thanks for the proposal. Sounds sensible to me, I'll create a ticket for it which you can follow :) have a happy new year!
